src/components/Customer/PPM/Closeout/AboutForm/AboutForm.stories.jsx

```
@@ -0,0 +1,113 @@
+  mtoShipment: {
+    id: 'f3c29ac7-823a-496a-90dd-b7ab0d4b0ece',
+    moveTaskOrderId: 'e9864ee5-56e7-401d-9a7b-a5ea9a83bdea',
+    shipmentType: SHIPMENT_OPTIONS.PPM,
+    ppmShipment: {
+      actualMoveDate: '2022-05-19',
+      actualPickupPostalCode: '10001',
+      actualDestinationPostalCode: '60652',
+      hasReceivedAdvance: false,
+    },
+  },
+  postalCodeValidator: () => {},
+};
+
+export const OptionalValues = Template.bind({});
+OptionalValues.args = {
+  mtoShipment: {
+    id: 'f3c29ac7-823a-496a-90dd-b7ab0d4b0ece',
+    moveTaskOrderId: 'e9864ee5-56e7-401d-9a7b-a5ea9a83bdea',
+    shipmentType: SHIPMENT_OPTIONS.PPM,
+    ppmShipment: {
+      actualMoveDate: '2022-05-19',
+      actualPickupPostalCode: '10001',
+      actualDestinationPostalCode: '60652',
+      hasReceivedAdvance: true,
+      advanceAmountReceived: 456700,
+    },
+  },
+  postalCodeValidator: () => {},
+};
+
+export const RequiredErrors = Template.bind({});
+RequiredErrors.args = {
+  mtoShipment: {
+    id: 'f3c29ac7-823a-496a-90dd-b7ab0d4b0ece',
+    moveTaskOrderId: 'e9864ee5-56e7-401d-9a7b-a5ea9a83bdea',
+    shipmentType: SHIPMENT_OPTIONS.PPM,
+    ppmShipment: {
+      hasReceivedAdvance: true,
+    },
+  },
+  postalCodeValidator: () => {},
+};
+
+RequiredErrors.play = async ({ canvasElement }) => {
+  // Starts querying the component from its root element
+  const canvas = within(canvasElement);
+
+  await userEvent.click(canvas.getByText('Save & Continue'));
+};
+
+export const InvalidZIPs = Template.bind({});
+InvalidZIPs.storyName = 'Invalid ZIPs';
+InvalidZIPs.args = {
+  mtoShipment: {
+    id: 'f3c29ac7-823a-496a-90dd-b7ab0d4b0ece',
+    moveTaskOrderId: 'e9864ee5-56e7-401d-9a7b-a5ea9a83bdea',
+    shipmentType: SHIPMENT_OPTIONS.PPM,
+    ppmShipment: {
+      actualMoveDate: '2022-05-23',
+      actualPickupPostalCode: '10000',
+      actualDestinationPostalCode: '20000',
+    },
+  },
+  postalCodeValidator: () => UnsupportedZipCodePPMErrorMsg,
```

**thought**: would it be good use `UnsupportedZipCodePPMErrorMsg` in all the stories? I was confused for a second why invalid zips weren't being flagged in the blank version. This isn't a blocker though, I'm fine with it either way.

---

You are talking like a mock implementation that has a valid and invalid dummy data that we'd need to know to get an error?